app/src/main/java/com/poupa/vinylmusicplayer/service/MusicService.java

```diff
@@ -619,6 +620,35 @@ public void onResourceReady(@NonNull Bitmap resource, Transition<? super Bitmap>
         }
     }
 
+    public void updateMediaSessionLyric(String line) {
+        final Song song = getCurrentSong();
+
+        if (song.id == -1) {
+            mediaSession.setMetadata(null);
+            return;
+        }
+
+        List<String> splitlist = StringUtil.wideCharacterSplit(line,20);
+
+        // set Artist, Title and Album to lyrics
+        final MediaMetadataCompat.Builder metaData = new MediaMetadataCompat.Builder()
```

Comment:
I dont get it: we are not sending lyrics here.

And if I'm not wrong, there is already similar code that publishes the now-playing metadata for the other apps (Android Auto, musixmatch, lock screen, ...)

Reply:
Are you trying to use the artist/title/album to convey the first 3 lines of the synchronized lyrics?

Reply:
Yes, I get the idea from several apps(like netease-music and qq-music) but they only use artist as lyric, so they cannot display full lyrics.